nixos/modules/services/hardware/upower.nix

```
@@ -179,24 +179,59 @@ in
         '';
       };
 
+      allowRiskyCriticalPowerAction = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Enable the risky critical power actions "Suspend" and "Ignore".
+        '';
+      };
+
       criticalPowerAction = lib.mkOption {
-        type = lib.types.enum [ "PowerOff" "Hibernate" "HybridSleep" ];
+        type = lib.types.enum [
+          "PowerOff"
+          "Hibernate"
+          "HybridSleep"
+          "Suspend"
+          "Ignore"
+        ];
         default = "HybridSleep";
         description = ''
           The action to take when `timeAction` or
           `percentageAction` has been reached for the batteries
-          (UPS or laptop batteries) supplying the computer
+          (UPS or laptop batteries) supplying the computer.
+
+          When set to `Suspend` or `Ignore`, `allowRiskyCriticalPowerAction`
```

I recently learned there's special syntax for referencing options. See https://github.com/NixOS/nixpkgs/tree/master/doc#roles

```suggestion
          When set to `Suspend` or `Ignore`, {option}`services.upower.allowRiskyCriticalPowerAction`
```

---

Good to know! Done.